uber/tests/models/test_attendee.py

```diff
@@ -260,6 +260,12 @@ def unset_volunteering(self, monkeypatch):
         monkeypatch.setattr(Attendee, 'unset_volunteering', Mock())
         return Attendee.unset_volunteering
 
+    @pytest.fixture(autouse=True)
+    def prevent_presave_adjustments(self, monkeypatch):
+        """ Prevent some tests from crashing on exit by not invoking presave_adjustements() """
+        monkeypatch.setattr(Attendee, 'presave_adjustments', Mock())
+        return Attendee.presave_adjustments
```

Comment:
There's no need for the ``return`` statement here, since we're not actually using the value provided by this fixture anywhere.  Not that it hurts anything to keep it in, just a superfluous line of code.

Also, to be super-nitpicky, this mock doesn't cause us to "not invoke presave_adjustments", it merely replaces ``presave_adjustments`` with a do-nothing function.

I wouldn't bother taking the effort to change either of these things though, even if this hadn't already been merged.